RealmModelGenerator/AttributeDetailVC.swift

```diff
@@ -24,28 +24,88 @@ class AttributeDetailVC: NSViewController, AttributeDetailViewDelegate, Observer
         }
     }
     
+    //MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
     }
     
     func invalidateViews() {
         if !self.viewLoaded || attribute == nil { return }
-        attributeDetailView.name = attribute!.name
```

Comment:
using a guard statement would also make this a lot more pretty b/c you won't need to use self.attribute!.someVar all the time